### ISSUE TITLE
Generation of namespaced action is broken on 1.1.0.beta1

### ISSUE THREAD

**Issue by gizotti** [hanami/hanami#833]
I was trying out `v1.1.0.beta1` and notice that the creation of namespaced actions are broken in the new CLI rewrite.

#### Shell Output
```shell
➜ bundle exec hanami g action api api/v1/posts#create
      create  spec/api/controllers/api/v1/posts/create_spec.rb
      create  apps/api/controllers/api/v1/posts/create.rb
      create  apps/api/views/api/v1/posts/create.rb
      create  apps/api/templates/api/v1/posts/create.html.haml
      create  spec/api/views/api/v1/posts/create_spec.rb
      insert  apps/api/config/routes.rb
```
#### View Created
```ruby
module Api::Views::Api::V1::Posts
  class Create
    include Api::View
  end
end
```

#### Shell Output
```shell
➜ bundle exec hanami g action api api/v1/posts#create
      create  apps/api/controllers/api/v1.rb
      create  apps/api/views/api/v1.rb
      create  apps/api/templates/api/v1.html.haml
      create  spec/api/controllers/api/v1_spec.rb
      create  spec/api/views/api/v1_spec.rb
```

#### View Created
```shell
module Api::Views::Api
  class V1
    include Api::View
  end
end
```

This is happening in [this method](https://github.com/hanami/hanami/blob/develop/lib/hanami/cli/commands/generate/action.rb#L56). The regex splits the `name` properly, but it assumes that the `action` won't be namespaced.

```shell
irb(main):006:0> controller, action = "api/v1/blah#create".split(/#|\//)
=> ["api", "v1", "blah", "create"]
irb(main):007:0> controller
=> "api"
irb(main):008:0> action
=> "v1"
irb(main):009:0>
```

Could you please assign this issue to me? I can get this done in the next few days.

**Comment by gizotti** [comment]
Had a deeper look at it and it's more than what I assumed in the description. I'll get a first draft in the next few days for review.

**Comment by mereghost** [comment]
Done, @gizotti . If you need any help, do not hesitate to ping @hanami/core =)

**Comment by jodosha** [comment]
Fixed by #835
